src/DocRaptor.php

```diff
@@ -185,12 +191,14 @@ public function fetchDocument($filename = false)
             curl_setopt($ch, CURLOPT_POST, count($fields));
             curl_setopt($ch, CURLOPT_POSTFIELDS, $fields_string);
             curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+            $result = curl_exec($ch);
 
-            if ($result = curl_exec($ch)) {
+            if ($result) {
                 $http_code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
 
                 if ($http_code != 200) {
-                    return False;
+                    //@todo silently erroring out is probably not the best way here
```

Comment:
As in other cases I would throw an exception with the status code and message from DR then consumer can deal with it. Printing just forces them to extend or core hack.